attachment-viewer/src/main/java/im/vector/lib/attachmentviewer/VideoLoaderTarget.kt

```diff
@@ -34,7 +34,8 @@ interface VideoLoaderTarget {
 
     fun onVideoFileLoading(uid: String)
     fun onVideoFileLoadFailed(uid: String)
-    fun onVideoFileReady(uid: String, file: File)
+    fun onVideoURLReady(uid: String, file: File)
```

Comment:
Why renaming the method?

Reply:
sh*t it renamed both function. meant to only renamed one